attachment_s3/models/ir_attachment.py

```diff
@@ -63,19 +64,6 @@ def _get_s3_bucket(self, name=None):
             params['region_name'] = region_name
         else:
             connect_s3 = boto.connect_s3
-        if not (access_key and secret_key and bucket_name):
```

Comment:
you cannot trash this so easily :)
Options:
1. override `_get_s3_bucket` in your own code. Easy and fast. No clash w/ us.
2. if you think this is something worth to support generically, you could add an extra env var like `AWS_IAM` or similar. Then if this var is present you bypass this part.

Reply:
Option 1, I will override ```_get_s3_bucket``` ;)